.github/CODEOWNERS

```diff
@@ -62,7 +62,7 @@
 /configs/default/FLCO-* @tobeornottobe81 @betaflight/unified-target-administrators
 /configs/default/FLLF-* @flyinglemonfpv @betaflight/unified-target-administrators
 /configs/default/FLMO-* @cvetaevvitaliy @betaflight/unified-target-administrators
-/configs/default/FOXE-* @Johnsnow-foxeer @betaflight/unified-target-administrators
```

Comment:
Just did some background investigation.

This config is exported from a third-party betaflight fork and should never be merged into betaflight official unified target, and iFlight should bear the responsibility of distribution on their own.

Although I don't mind being mentioned, but since you considered yourself as a senior engineer of iFlight, then please do not submit such an amateur PR.

Reply:
Well, if there would be a proper documentation besides those 77 links and useless 99 pages of tutorial that suddenly end with "Pull your request here" maybe some more senior engineers would not need to read your amateur answers. Sort out your mood swings and provide a step by step tutorial on how to actually submit a target, for those stupid senior engineers with pictures maybe... ;) I've been in contact with MK already!

Reply:
> We've relocated production and target responsibility.

This is your iFlight home affair, nothing related to us. But for other maintainer's interest, I have already received confirmation from fab that the "new FC" you claimed has been discontinued the year before last year. You are just over-marketing for those old FC keeping in stock.

> Turns out, I still just wanna submit a target and still don't wanna learn how to code.

If you don’t want to code and still want to submit a new target, you can kindly ask how to do it, or you should hire some professional programmers so as not to waste all of ours time. Your monster PR has already wasted my 10 minutes in the morning, though I don’t want to blame you for it. 

> Well and, not everybody is a Coder... Did you know?! If there would be a proper documentation besides those 77 links and useless 99 pages of tutorial that suddenly end with "Pull your request here" maybe some more senior engineers would not need to read your amateur answers. Checked some YouTube content and the provided information. ...... Sort out your mood swings...

Complaining about the documentation is not the right approach. If you are dissatisfied with our documentation, you can help us improve it. We don’t need you to come here to criticize us. Frankly speaking, I don't think I have any mood swings, and I can't understand your allegations of so-called "mood swings" against me.

> ... and provide a step by step tutorial on how to actually submit a target, for those stupid senior engineers with pictures maybe... ;)

Sorry but I refuse to teach you so just for your bad attitude, this trick might work in iFlight, but not on me.

> Betaflight / STM32F405 (S405) 4.2.9 Apr 27 2021 / 19:33:01 (e097f4ab7) MSP API: 1.43

Returning to the product itself, I have successfully traced the source of the code based on the code commit you provided. It turns out that this version comes from a third-party betaflight fork: https://github.com/mbilozub/fpv_custom_mods/tree/master/mods/dji-warning/betaflight/4.2.9, and I think we have already stated our position on this DJI OSD related repo.

> I've been in contact with MK already!

It happens that the statement was sent by @mikeller and passed on to DJI by me, so don't wish Mike to come out to refute his own views. You are just wasting everybody's time.

Reply:
![WeChat Image_20210821150217](https://user-images.githubusercontent.com/31681897/130313709-7fc4eb51-dfdc-41da-9f58-6a0e869bd4ec.png)

Reply:
`This is your iFlight home affair, nothing related to us. But for other maintainer's interest, I have already received confirmation from fab that the "new FC" you claimed has been discontinued the year before last year. You are just over-marketing for those old FC keeping in stock.`
I've been told it is difficult to work with you and I'm not disappointed of the outcome. This is a new FC, just made in an old factory. Are you involved in competing products btw? Your chinese social media almost looks like it.

`If you don’t want to code and still want to submit a new target, you can kindly ask how to do it, or you should hire some professional programmers so as not to waste all of ours time. Your monster PR has already wasted my 10 minutes in the morning, though I don’t want to blame you for it.`
That was my first post about trying to contact MK and making sure what I've done wrong can be fixed or explained to me before being called an Amateur by you. Not sure what rivals you have with iFlight, but your attitude towards us is well known on our side!

`Complaining about the documentation is not the right approach. If you are dissatisfied with our documentation, you can help us improve it. We don’t need you to come here to criticize us. Frankly speaking, I don't think I have any mood swings, and I can't understand your allegations of so-called "mood swings" against me.`
Happy to help, pointing out there documentation stops with "Create a Pull Request" and then you need to be a Github Coder or in the Chinese circle of target uploaders to know how it works. I tried to educate myself on how to copy the fork and create my own branch, but decided to just post the dump to see what reaction I get first.

`Sorry but I refuse to teach you so just for your bad attitude, this trick might work in iFlight, but not on me.`
This is just unprofessional and doesn't need any comment!

`Returning to the product itself, I have successfully traced the source of the code based on the code commit you provided. It turns out that this version comes from a third-party betaflight fork: https://github.com/mbilozub/fpv_custom_mods/tree/master/mods/dji-warning/betaflight/4.2.9, and I think we have already stated our position on this DJI OSD related repo.`
My engineer made a mistake obviously, using the wrong target to export the diff all bare to me.

I think this conversation with you is not going anywhere! I'll keep in touch with MK to work on future projects as well as making sure the target submission works well. Wish you Chinese Devs would be less involved in that chinese competition drama nowadays, but well... I see where this is going.